src/main/java/com/example/lesson6/zad3/EvenOrOddNumbers.java

```diff
@@ -0,0 +1,19 @@
+package com.example.lesson6.zad3;
+
+public class EvenOrOddNumbers {
+
+
+    public static boolean getEvenNumbers(int number){
```

Comment:
metoda zwraca booleana, powinna zaczynać się od is, czyli wystarczy np. isEvenNumber albo samo isEven